themes/src/main/resources-community/theme/base/login/messages/messages_pt_BR.properties

```diff
@@ -202,13 +264,108 @@ invalidCodeMessage=C\u00F3digo inv\u00E1lido, por favor fa\u00E7a login novament
 identityProviderUnexpectedErrorMessage=Erro inesperado durante a autentica\u00E7\u00E3o com o provedor de identidade
 identityProviderNotFoundMessage=N\u00E3o foi poss\u00EDvel encontrar um provedor de identidade com o identificador.
 identityProviderLinkSuccess=Sua conta foi vinculada com sucesso com {0} conta {1} .
-realmSupportsNoCredentialsMessage=O realm n\u00E3o suporta qualquer tipo de credencial.
-identityProviderNotUniqueMessage=O realm suporta m\u00FAltiplos provedores de identidade. N\u00E3o foi poss\u00EDvel determinar qual o provedor de identidade deve ser usado para se autenticar.
+staleCodeMessage=Esta p\u00E1gina n\u00E3o \u00E9 mais v\u00E1lida. Volte ao seu aplicativo e fa\u00E7a login novamente
+realmSupportsNoCredentialsMessage=O dom\u00EDnio n\u00E3o suporta qualquer tipo de credencial.
+credentialSetupRequired=N\u00E3o \u00E9 poss\u00EDvel fazer o login, \u00E9 necess\u00E1ria configura\u00E7\u00E3o de credencial.
+identityProviderNotUniqueMessage=O dom\u00EDnio suporta m\u00FAltiplos provedores de identidade. N\u00E3o foi poss\u00EDvel determinar qual o provedor de identidade deve ser usado para se autenticar.
 emailVerifiedMessage=O seu endere\u00E7o de e-mail foi confirmado.
+staleEmailVerificationLink=O link em que voc\u00EA clicou \u00E9 um link antigo e n\u00E3o \u00E9 mais v\u00E1lido. Talvez voc\u00EA j\u00E1 tenha verificado seu e-mail.
+identityProviderAlreadyLinkedMessage=A identidade federada retornada por {0} j\u00E1 est\u00E1 vinculada a outro usu\u00E1rio.
```

Comment:
To technical? Maybe`A conta retornada do {0} já está vinculada ...` ?

Reply:
Ok